data-ai/anomaly-detection.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Anomaly Detection with Azure Databricks
+category: Machine Learning, Advanced Analytics, AI, Data
+date: 2018-12-30
+tags: [Anomaly Detection, Predictive Modeling, Machine Learning, Advanced Analytics, AI, Data, Databricks, Azure, Spark, ML Pipelines, Random Forests]
```

Comment:
Tags should be be all lower case, no spaces (hypens only) and limited of a maximum of FOUR per post. Do not use tags that just duplicate the category such as "ai" or "data", people will mostly be looking for azure service names so try to use those where possible, check existing tags to make sure you're not unnecessarily introducing new tags, or tags for the same thing but with slightly different name (e.g. "azure-logicapp" vs "logicapp")